src/param.F

```diff
@@ -228,8 +228,8 @@ subroutine param(namelist_input,                                  &
 #endif
 
       call bcast_log(terrain_flag,device=.true.)
-      call bcast_real(dx)
-      call bcast_real(dy)
+      call bcast_real(dx,device=.true.)
+      call bcast_real(dy,device=.true.)
```

Comment:
My bad. I did not realize that these two variables are now used in an OpenACC subroutine and should be updated manually. Thanks for catching this error!

Reply:
@ no problem.  I was a bit surprised why the compiler did not flag it as a fatal error.

Reply:
Probably because `dx` and `dy` are used in both acc routine and acc kernel?